common/lighthouse_version/src/lib.rs

```diff
@@ -1,6 +1,9 @@
 use git_version::git_version;
 use target_info::Target;
 
+/// Returns the version number.
+pub const VERSION_NUMBER: &str = "1.1.3";
```

Comment:
Is this supposed to be here?

Reply:
Not sure why I added this. will remove.

Reply:
Actually, using this to return the `client_version` parameter. Forgot to update it.